backend/capellacollab/sessions/files/routes.py

```diff
@@ -90,3 +91,23 @@ def upload_files(
     OPERATOR.upload_files(id, tar_bytes)
 
     return {"message": "Upload successful"}
+
+
+@router.get(
+    "/download",
+)
+def download_file(
+    id: str,
+    filename: str,
+    db: Session = Depends(get_db),
+    token=Depends(JWTBearer()),
+) -> UploadFile:
+    check_session_belongs_to_user(get_username(token), id, db)
+
+    return StreamingResponse(
+        OPERATOR.download_file(id, filename),
+        headers={
+            "content-disposition": 'attachment; filename="download.zip"',
```

Comment:
```suggestion
            "content-disposition": 'attachment; filename=f"{filename}.zip"',
```

Can we use the real filename here?